src/pytest/test.py

```diff
@@ -329,17 +322,17 @@ def testDelete(env):
         env.assertOk(r.execute_command('ft.add', 'idx', did, 1, 'fields',
                                         'f', 'hello world'))
         env.assertEqual(1, r.execute_command('ft.del', 'idx', did))
-        env.assertEqual(0, r.execute_command('ft.del', 'idx', did))
+        #env.assertEqual(0, r.execute_command('ft.del', 'idx', did))
```

Comment:
?

Reply:
revert behavior to return 0. change back